src/bindings/html/service.js

```diff
@@ -16,7 +16,6 @@ export function initViews(langs) {
 
 function initView(view, langs) {
   dispatchEvent(view.doc, 'supportedlanguageschange', langs);
-  return view.ctx.fetch(langs, 1).then(
-    translateDocument.bind(view, view.doc, langs)).then(
-      () => view.observe());
+  view.ctx.fetch(langs, 1).then(
```

Comment:
Please return the promise here.  Otherwise the `Promise.all` in `initViews` doesn't do anything useful.  (We're not taking advantage of it right now, but I think the code is cleaner with it.)